tests/nest_tests/resources/nestml_allmodels_codegen_opts.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Could you rename this file to include `nest_2` somewhere since this is for the NEST2 target?